www/js/common/factories/group.factory.js

```diff
@@ -1,123 +1,85 @@
-'use strict';
-
-angular.module('main').factory('GroupFactory', ['$q', '$rootScope', function ($q, $rootScope) {
+angular.module('main').factory('GroupFactory', ['$q', '$rootScope', 'AuthService', '$firebaseObject', '$firebaseArray', function ($q, $rootScope, AuthService, $firebaseObject, $firebaseArray) {
```

Comment:
You might check out a build utility called ng-annotate, it unburdens you of this "long hand" injection notation. [See here](https://github.com/olov/ng-annotate).